style.css

```diff
@@ -32,25 +32,6 @@
    line-height: 1.5em;
  }
  
- .default {
-   padding: 1rem 2rem;
- }
- 
- .image {
-   padding: 0;
-   margin: 0;
-   font-size: 0;
- }
- 
- .image img {
-   width: 100%;
- }
- 
- .default img {
-   max-width: 100%;
```

Comment:
This one has a big impact: all content images had a `max-width: 100%` before that is not there anymore.

Reply:
Ok, but it only affected images within embeds, right? I couldn't spot a difference in rendering of normal article images.

Reply:
Yes. I just raised the concern, there might be some cases we do not think of...